### PR TITLE
add pytest-copie package

### PR DESCRIPTION
**Pull request by 12rambau**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by 12rambau** [comment]
@conda-forge/help-python

this is not the first time I package a lib that have entry point but as this one is a pytest plugin it's a bit different as it has 2 levels. 

```toml
[project.entry-points."pytest11"]
copie = "pytest_copie.plugin"
```

How should I write it down in the `build` section of the recipe? 

thanks for your help.

File: recipes/pytest-copie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest_copie-{{ version }}.tar.gz
+  sha256: 7b3182af78af35d5724dd26dbb2fca1d7868c13fe4326262b3ce7b5ab8159f7e
+
+build:
+  entry_points:
+
```
Comment (ocefpaf) [inline recipes/pytest-copie/meta.yaml:14]:
```suggestion
```

**Comment by ocefpaf** [comment]
> How should I write it down in the `build` section of the recipe?

I don't think you need anything for this kind of entry point. If I understand it correctly pytest will find this package and add the proper call to its CLI, so commands like `pytest --keep-copied-projects` will work. Let's give it a go and let me know if that doesn't work.

**Comment by 12rambau** [comment]
Is there a way for me to test it before it's released in a feedstock ?

**Comment by ocefpaf** [comment]
> Is there a way for me to test it before it's released in a feedstock ?

Yep. I don't know the library but you can create a test case and call the CLI or a dummy example. Also, running the full test suit is fair game if you have them in the source distribution.

**Comment by 12rambau** [comment]
Ok I'm stupid, the tests folder is not part of the distribution. In any case, the last failed run actually show that the plugin is part of the pytest run so I'm good. let me roll back to previous stage.

**Comment by 12rambau** [comment]
Thanks for your fast answer !

**Pull request merged**